Logic Problem.md

```diff
@@ -0,0 +1,37 @@
+# Summary
+> >This task examined how different coordination demands and how individual contributations could be combined into a group product. To do this, participants performed anagram and logic tasks.
+
+# References
+>Main paper: https://www.researchgate.net/profile/Glenn-Littlepage/publication/247746082_Effects_of_Group_Size_and_Task_Characteristics_on_Group_Performance_A_Test_of_Steiner%27s_Model/links/59073d83a6fdccd580d5debd/Effects-of-Group-Size-and-Task-Characteristics-on-Group-Performance-A-Test-of-Steiners-Model.pdf
+> Logic Problems: Whipkey, R. L. (1982). Skiing vacations. In Moore, R., Fothstein, E., Reineke, K. (Eds.), The Dell big book of crosswords and pencil puzzles #2 (p. 157). New York, NY: Dell.
+
+
+## Materials for alternative versions of the experiment 
+>Three variations of each type of task was used to prevent subjects in one group from communicating to another group
+
+# Procedure
+## Steps
+> Randomly assign participants to work alone or in groups of 2, 5, or 10.
+> 
+> First, participants will complete 33 anagrams randing from 5 to 14 letters in 5 minutes
+> 
+> Then, participants with complete a logic problem by compiling information among the different group members and make 15 deductions to arrive at a solution in 25 minutes.
+> > An example of a logic problem:
+> > > 1. Pair first and last names with destinations to determine which five couples vacationed at various ski resorts
+> > > 2. Clues include: "Neither Tammy nor Sue vacationed in Innsbruck," and Both Rita and Mrs. Owens returned from their vacations with broken wrists."
```

Comment:
clues were distributed among different group members so that communication was required since clues had to be combined